join.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: translated
+permalink: /join
+namespace: join
+---
+# {% t connect.info %}
+{% include links.html %}
```

Comment:
Should include a privacy statement, e.g, `https://utaw.tech/privacy` has good template to be GDPR/DSGVO compliant